.github/workflows/build-wheels.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           echo $release_tag
 
           # Install dependencies.
-          pip install .
+          cd apis/python && pip install . && cd ../..
```

Comment:
Was this a merge conflict? I believe we need to keep this as it was.

Reply:
https://github.com/dudoslav/TileDB-Vector-Search/actions/runs/7916707804/job/21611139539#step:6:7

but it seems that it works

Reply:
I think that that pipeline would not work on release

Reply:
Returned